tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -170,35 +176,6 @@ jobs:
       artifactName: 'ContribOperators.md'
 
 
-  - task: PythonScript@0
-    displayName: 'Regenerate cmake config with STATIC_ANALYSIS=ON '
```

Comment:
This part is temporarily removed because there is a bug in the latest VC++ release.

Reply:
nit: if we want to eventually add this back (after the bug fix is available) you can turn this step off by using  condition False ... this way once the fix is available you can simply enable it again...